roles/datadog/tasks/main.yml

```diff
@@ -10,7 +10,8 @@
         - "env:{{ rails_env }}"
         - "host-id:{{ host_id | default(ansible_limit) }}"
       logs_enabled: true
-      apm_enabled: "{{ enable_rails_apm | default('false') }}"
```

Comment:
I'm not sure about this change. `apm_enabled` is the key used in the datadog role.

Reply:
@sauloperez Is it possible you're using a newer version of `Datadog.datadog`? The README for version `2.4.0` specified in `bin/requirements.yml` does not mention `apm_config`.